lib/util/date.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+function formatHMS(int $seconds): string
+{
+    return sprintf("%d:%02d:%02d", $seconds / 3600, ($seconds / 60) % 60, $seconds % 60);
```

Comment:
Haven't tested it but Carbon could do that:

```php
CarbonInterval::seconds($seconds)->cascade()->forHumans();
```

Reply:
That particular command generates something too verbose for the placement in the grid:

![image](https://user-images.githubusercontent.com/32680403/210889574-b1073683-5b22-4955-9afe-a2fbb1d1f26d.png)

Reply:
Checking the docs it seem `->forHumans(['short' => true])` can make it less verbose.

However, sticking with our own formatting is just fine imo. Just wanted to point out that carbon can do a lot of those things :)